mrjones2014/yaclt
# Make changeType tags optional by removing the Handlebars variable fro…

…m the format string

Resolves: #85 

## How to Test

1. in `yacltrc.js` change `validationPattern` from `"\\[.*\\]\\s+.*\\s{#[\\d]+}",` to `"*\\s{#[\\d]+}",`, and change `format` to `'{{message}} {{append "" "{"}}#{{issueId}}{{append "" "}"}}\n'`
1. Run `yarn build; ./dist/index.js new -m "test"` it should generate a new changelog without a change type tag
1. Run `./dist/index.js validate` the validation should pass

## Potential Side Effects/Things to Look Out For

None expected
